src/tools/create-app-request-client.js

```diff
@@ -41,6 +42,11 @@ const createAppRequestClient = (input, options) => {
 
   httpBefores.push(addQueryParams);
 
+  const verify = _.get(input, '_zapier.event.verify');
```

Comment:
I'd only #suggest renaming this to `verifySSL`, `verifySsl`, or `veryifyCertificate`.